src/docs/README.md

```diff
@@ -13,30 +13,26 @@ todo:
 <br>
 Your contribution to the DataPLANT knowledge base is highly appreciated.
 This guide is intended to show you how to contribute new articles and tutorials or review and adapt parts of existing ones.
-<!-- most efficiently for you and the knowledge base curators -->
 
-<!-- TODO check link!! -->
 For smaller changes and suggestions, please refer to [here](./home.html).
 
 
 - [Introduction](#introduction)
 - [GitHub Routines](#github-routines)
 - [Testing](#testing)
-- [Common errors](#common-errors)
-- [Authoring Content](#authoring-content)
-    - [Docs-Page](#docs-page)
-    - [_sidebar](#_sidebar)
-    - [Markdown to HTML Rendering Rules](#markdown-to-html-rendering-rules)
-        - [Ignored Content](#ignored-content)
-        - [Visible or Hidden](#visible-or-hidden)
-- [Content Design Principles](content-design-principles)
-    - [References](#references)
-    - [Structure and Format](#structure-and-format)
-    - [Images](#images)
-    - [File Name Requirements](#file-name-requirements)
-    - [Addressing Readers](#addressing-readers)
-    - [Language](#language)
-
+- [Common Errors](#common-errors)
```

Comment:
Why delete the h1 headers from TOC?

Reply:
That's my default of the vscode extension updating the TOC... 
I've updated the h1's to h2's, etc. to keep title as the only h1. 
The README will need some overall revision and re-structuring.

Reply:
> I've updated the h1's to h2's, etc. to keep title as the only h1.

I don't really see why we should do that. I think using h1-h3 is perfectly fine and creates nicely organized tocs.

But if you prefer it that way we can do that.

Reply:
Of course we can use h1, but then the first heading used should be an h1. 
The way it was, we had some h2's first and then at some point in the middle two h1's